app/src/main/res/values-ar/strings.xml

```diff
@@ -702,7 +702,7 @@
     <string name="signin_with_google_title">المصادقة لاستخدام Google Drive™</string>
     <string name="show_system_apps">عرض تطبيقات النظام</string>
     <string name="system_apps">System apps</string>
-    <string name="third_party_apps">Third party apps</string>
+    <string name="third_party_apps">User apps</string>
```

Comment:
Any string changes other than from values/strings.xml will be overwritten by the transifex pipeline. You should remove these changes.